MyLibrary/MyLibraryApp.Application/Services/Interfaces/ICategoryService.cs

```diff
@@ -0,0 +1,17 @@
+﻿using MyLibraryApp.Application.Models.Books;
+using MyLibraryApp.Application.Models.Categories;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MyLibraryApp.Application.Services.Interfaces
+{
+    public interface ICategoryService
+    {
+        Task<CategoryResponseModel> CreateCategoryAsync(CreateCategoryRequestModel requestModel);
+        Task<List<CategoryResponseModel>> ReadAllCAtegoriesAsync();
```

Comment:
Consistency again: the other three interfaces have an empty newline between methods declaration. This means that this interface should look similar too. So please add a newline between each method declaration. Also fix the `IUserService` one which has one small similar inconsistency.